### PR TITLE
Link swagger in navbar

### PR DESCRIPTION
**Pull request by pat-s**
(testing)

fix point 1 from #1944

### CODE REVIEW

**Comment by pat-s** [comment]
@anbraten Tagging you as I saw you edited the vue files at some point.

I wanted to add a link in the navbar pointing to the swagger UI interface. Couldn't quite get it working by just trying to bake something together without Vue knowledge. If you like the idea, it might be a small thing for you to finalize the idea?

**Comment by 6543** [comment]
@pat-s ref =  #1944

**Comment by pat-s** [comment]
The navbar item is there 

image


but the link is still faulty, for some reason it expands to `$INSTANCE/null/swagger/index.html` (the inserted `null` is the issue)

File: web/src/components/layout/header/Navbar.vue [hunk 1]
```diff
@@ -70,14 +72,15 @@ export default defineComponent({
     const authentication = useAuthentication();
     const { darkMode } = useDarkMode();
     const docsUrl = window.WOODPECKER_DOCS;
+    const apiUrl = `window.WOODPECKER_ROOT_URL/swagger/index.html`;
```
[the remark this answers is not in the document]
Comment (pat-s) [reply web/src/components/layout/header/Navbar.vue]: Nope, see comment above

[the remark this answers is not in the document]
Comment (pat-s) [reply web/src/components/layout/header/Navbar.vue]: I was referring to your suggesting from yesterday. The new one I didn't try yet -> it's building now.

**Comment by pat-s** [comment]
Working now!

File: web/src/components/layout/header/Navbar.vue [hunk 2]
```diff
@@ -70,14 +72,15 @@ export default defineComponent({
     const authentication = useAuthentication();
     const { darkMode } = useDarkMode();
     const docsUrl = window.WOODPECKER_DOCS;
+    const apiUrl = `${config.rootURL ?? ''}/swagger/index.html`;
```
Comment (6543) [inline web/src/components/layout/header/Navbar.vue:75]: See  #1992

I would merge mentioned pull first, and then refactor to use that config

**Review by 6543 — APPROVED** [review]

**Pull request merged**